spec/flow_extras_spec.lua

```diff
@@ -36,9 +46,9 @@ describe("List", function ()
 		assert.same(gui.Stack{
 			align_h = "center",
 			align_v = "center",
-			gui.VBox{ spacing = 0.25, },
-			gui.List{ inventory_location = "a", list_name = "b", w = 0, h = 0 }
-		}, flow_extras.List{ inventory_location = "a", list_name = "b", w = 0, h = 0 })
+			gui.Image{ bgimg = "flow_extras_list_bg.png", w = 1, h = 1 },
```

Comment:
All of these had to be 1x1 because 0x0 isn't valid, and wasn't possible anymore. Not breaking because it wasn't supported.